index.html

```diff
@@ -383,6 +382,35 @@
           <var>oldSubscription</var> and a <a>PushSubscription</a> instance representing the
           <a>push subscription</a> having the new keys as <var>newSubscription</var>.
         </p>
+        <p>
+          To <dfn>create a push subscription</dfn>, given an <a>PushSubscriptionOptions</a> object
+          of <var>options</var>, the <a>user agent</a> must run the following steps:
+        </p>
+        <ol>
+          <li>Let <var>subscription</var> be a new <a>push subscription</a>.
+          </li>
+          <li>Set the <code>options</code> attribute of <var>subscription</var> to be a copy of
+          <var>options</var>.
+          </li>
+          <li>Generate a new P-256 <a>ECDH</a> key pair. Store the private key in an internal slot
```

Comment:
I'm not sure how to phrase this. "as described in [[!X9.62]]"?

The actual curve parameters are defined in Section 10.2, should we refer that too?